de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUShipperTransportationBL.java

```diff
@@ -75,7 +75,7 @@
 		// Make sure Shipper Transportation is still open
 		if (shipperTransportation.isProcessed())
 		{
-			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@");
+			throw new AdempiereException("@M_ShipperTransportation_ID@: @Processed@=@Y@", new Object[] { shipperTransportation.getM_ShipperTransportation_ID() });
```

Comment:
i think it doesn't work like this..maybe you can clarify with teo..at any rate, i believe the M_ShipperTransportation_ID won't make it into the error message

Reply:
fixed